tests/integration/usage_statistics/usage_stats_event_examples.py

```diff
@@ -381,5 +381,6 @@
     "data_context_id": "2edf8eba-7de4-46fb-80b0-b8b7f7c7cd24",
     "data_context_instance_id": "cc57338c-44fc-4133-aed1-c7f5f8c577ac",
     "mac_address": "8422aebe6c3db9612f79d14c6b9280e65e53ef969db3aff4281e3035fb3ce86f",
+    "oss_id": None,
     "ge_version": "0.14.12+27.g7774964f6.dirty",
```

Comment:
most prod users are not cloning the repo. they're pip installing. so probably best to indicate a prod version like 0.16.15 instead of dirty branch